packages/input-container/src/vaadin-input-container-base-styles.js

```diff
@@ -61,6 +62,7 @@ export const inputContainerStyles = css`
       border-radius: 0;
       min-width: 0;
       font: inherit;
+      font-size: 1em;
```

Comment:
Is this only needed for text-area or do we want to have it for slotted inputs as well?

Reply:
It’s not absolutely required, but I thought it would be a good safeguard against inheriting the same em-based font size, if someone for some reason set the same `0.9em` size on both the ::part(input-field)` and the slotted `input`.

But now that I said that, perhaps we should actually be more strict about overriding all the styles of the slotted input with `!important` and make sure no CSS reset styles or other global styles mess them up?

Reply:
I think we can keep the current approach for now. IMO `!important` should be only used if changing some property breaks the component.